integrationTests/consensus/consensus_test.go

```diff
@@ -49,7 +49,7 @@ func initNodesAndTest(numNodes, consensusSize, numInvalid uint32, roundTime uint
 		for i := uint32(0); i < numInvalid; i++ {
 			nodes[i].blkProcessor.ProcessBlockCalled = func(blockChain data.ChainHandler, header data.HeaderHandler, body data.BodyHandler, haveTime func() time.Duration) error {
 				fmt.Println("process block invalid ", header.GetRound(), header.GetNonce(), getPkEncoded(nodes[i].pk))
```

Comment:
should we also change the logged message to match the returned error?

Reply:
I think that this print is something generic here and anyway the returned error will be printed in the caller side to spot the exactly error occurred